cloud-deployment-guide/editor-plugin-version.md

```diff
@@ -26,35 +26,35 @@ Support for requesting specific versions of {{site.productname}} {{site.productm
 
 ### dev, testing, and stable releases
 
-Choose from `dev`, `testing`, or `stable` release channels to load the latest version of {{site.productname}} from {{site.cloudname}}.
+Choose from `5-dev`, `5-testing`, or `5-stable` release channels to load the latest version of {{site.productname}} from {{site.cloudname}}.
```

Comment:
This is going to conflict with the other PR currently open to rework this isn't it? I'm also not sure how this is related to RTC, so it seems like this shouldn't be here.

If it is intentional, should this be using `5` instead of `5-stable` as the other PR changed that? Also should the order instead be `5`, `5-testing` and `5-dev` to since we expect customers to use `5`?

Reply:
Fix by revert https://github.com/tinymce/tinymce-docs/pull/2091/commits/0985bdd37731d79b2d2e06954a960fa2665e7587